### PR TITLE
Add fixtures for CV page

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/DataFixtures/ORM/LoadSummaryData.php [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        $summary1 = $summaryFactory->createFromArray([
+            'adherent' => $manager->getRepository(Adherent::class)->findByUuid(LoadAdherentData::ADHERENT_1_UUID),
+            'slug' => 'summary-michelle-dufour',
+            'availability' => JobDuration::PART_TIME,
+            'contactEmail' => 'michelle.dufour.2@example.ch',
+            'contributionWish' => Contribution::VOLUNTEER,
+            'jobLocation' => JobLocation::ON_REMOTE,
+            'motivation' => 'C\'est mon secret' ,
+            'professionalSynopsis' => '30 ans de travail dans le domaine scientifique',
+
```
Comment (HeahDude) [inline src/DataFixtures/ORM/LoadSummaryData.php]: Extra empty line.

File: src/DataFixtures/ORM/LoadSummaryData.php [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+use AppBundle\Summary\SummaryFactory;
+use Cocur\Slugify\Slugify;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class LoadSummaryData implements FixtureInterface, ContainerAwareInterface
+{
+    use ContainerAwareTrait;
```
Comment (HeahDude) [inline src/DataFixtures/ORM/LoadSummaryData.php]: Does not seem to be used.

File: src/Summary/SummaryFactory.php [hunk 3]
```diff
@@ -19,4 +19,26 @@ public function createFromAdherent(Adherent $adherent): Summary
     {
         return Summary::createFromMember($adherent, $this->slugger->slugify($adherent->getFullName()));
     }
+
+    public function createFromArray(array $data): Summary
```
Comment (HeahDude) [inline src/Summary/SummaryFactory.php]: Please add a comment to explicit that this method should be used for test purpose only.

File: src/Summary/SummaryFactory.php [hunk 4]
```diff
@@ -19,4 +19,33 @@ public function createFromAdherent(Adherent $adherent): Summary
     {
         return Summary::createFromMember($adherent, $this->slugger->slugify($adherent->getFullName()));
     }
+
+    /**
+     * Used for test and loadiang fixtures purpose.
```
Comment (HeahDude) [inline src/Summary/SummaryFactory.php]: Typo: loading. Phpdocs can be removed.

File: src/DataFixtures/ORM/LoadSummaryData.php [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+            'websiteUrl' => 'https://michelle-dufour-fake.en-marche-dev.fr',
+            'viadeoUrl' => 'https://fr.viadeo.com/michelle-dufour-fake',
+            'facebookUr' => 'https://www.facebook.com/michelle-dufour-fake',
+            'twitterNickname' => 'https://twitter.com/michelle-dufour-fake',
+            'missionWishes' => $manager->getRepository(MissionType::class)->findBy(['name' => 'Missions de bénévolat']),
+        ]);
+        $manager->persist($summary1);
+
+        // Experiances
+        $experiance11 = new JobExperience();
```
Comment (Stoakes) [inline src/DataFixtures/ORM/LoadSummaryData.php]: Hello,

experience is mispelled. But maybe there is a reason for that.

Cheers

File: src/DataFixtures/ORM/LoadSummaryData.php [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+            'currentProfession' => 'Bio-informaticien',
+            'websiteUrl' => 'https://michelle-dufour-fake.en-marche-dev.fr',
+            'viadeoUrl' => 'https://fr.viadeo.com/michelle-dufour-fake',
+            'facebookUr' => 'https://www.facebook.com/michelle-dufour-fake',
+            'twitterNickname' => 'https://twitter.com/michelle-dufour-fake',
+            'missionWishes' => $manager->getRepository(MissionType::class)->findBy(['name' => 'Missions de bénévolat']),
+        ]);
+        $manager->persist($summary1);
+
+        // Experiances
```
Comment (tgalopin) [inline src/DataFixtures/ORM/LoadSummaryData.php]: Typo

**Pull request merged**